### DISCUSSION TITLE
6월 블로그 개선팀

### DISCUSSION THREAD

**Discussion by jho2301** [woowacourse/tecoble#383] [General]
### 활동기간

6.1 ~ 6.30

### 팀원

FE: 미키
BE: 완태, 와이비

### 할 일 목록

1. 개선 사항 리스트업
2. 일정 산출
3. 개선 진행
4. 회의 시, 진행사항 브리핑

**Comment by jho2301** [comment]
## 6월 개선 및 추가 기능 목표

- TeCoBle로 로고 리뉴얼
- 3기 중심으로 사진 교체
- GA(Google Analytics)를 통한 방문자 통계 추가
- 검색 기능 추가

## 6월 초기 일정

### 6.3. ~ 6.6. 

각자 구조 파악, 로고 수정 초안과 리뉴얼 사진 후보 선정

### 6.7. 

#### 블로그 개선 팀 크루 자체 오전 회의

- 테코블 사진후보 선정
- 기능에 대한 논의 및 6월 전체 일정 픽스

#### 글쓰기 팀 본 회의

- 테코블 사진 후보 선정 완료
- 추가할 기능에 대한 개요 및 방향 , 타임라인 발표
- 기능에 대한 추가 건의 사항 수합

#### 글쓰기 팀 본 회의 종료 후

- 로고 및 바뀐 사진을 블로그에 반영

**Comment by kang-hyungu** [comment]
- 사이트 리뉴얼
  - 사진 수합 (미키, 와이비, 완태) (~06.11)
  - 사진 전환 기능 (미키 ) (~06.21)
  - 메인페이지 NavigationBar 추가 (와이비) (~06.30)
  - javable -> tecoble 단어 및 로고 이전 (완태) (~06.21)
  - javable redirect to tecoble (완태) (~06.21)
  - 내부 링크 수정 (완태) (~06.30)
- 태그 기능
  - 태그 스럽게 글자 크기 키우기 + # 붙이기 (와이비) (~06.15)
  - 태그 모아보기 (미키) (~06.30)
- 통계 (우선 순위 후순위)
  - 구글 Analytics (TBD) (~06.30)
